app/schema/geojsonSchema.ts

```diff
@@ -0,0 +1,14 @@
+import { z } from "zod";
+
+const coordinateSchema = z.array(z.number()).length(2);
+
+export const multiPolygonSchema = z.object({
+  type: z.literal("MultiPolygon"),
+  crs: z
+    .object({
+      type: z.string(),
+      properties: z.object({}),
+    })
+    .optional(),
+  coordinates: z.array(z.array(z.array(coordinateSchema))),
+});
```

Comment:
if we were going to define more geojson types i would go searching for a library that has already solved this problem